src/main/kotlin/com/goodchoice/domain/review/model/OwnReview.kt

```diff
@@ -0,0 +1,20 @@
+package com.goodchoice.domain.review.model
+
+import com.goodchoice.domain.common.model.Reference
+import com.goodchoice.domain.image.model.ImageDescriptor
+import com.goodchoice.domain.subject.model.Mark
+import com.goodchoice.domain.user.model.UserPreview
+import java.util.*
+
+data class OwnReview(
+    val id: UUID,
+    val title: String,
+    val subject: Reference,
+    val author: UserPreview,
+    val mark: Mark,
+    val advantages: List<String>,
+    val disadvantages: List<String>,
+    val bodies: List<ReviewBody>,
+    val votes: ReviewVotes,
```

Comment:
I think reviewer should be able to vote own review. It is a common approach in all social networks, for example. So, we can avoid `ReviewVotesWithOwn` creation